examples/python/l1-resources/project.py

```diff
@@ -0,0 +1,24 @@
+from aws_cdk import (
+  # Duration,
+  Stack,
+  # aws_sqs as sqs,
+)
+from constructs import Construct
+
+from awscdk_resources_mongodbatlas import CfnProject
+
+class CdkTestAppPython2Stack(Stack):
+
+  def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+    super().__init__(scope, construct_id, **kwargs)
+
+    project_name = self.node.try_get_context('project_name')
+    org_id = self.node.try_get_context('org_id')
+    profile_name = self.node.try_get_context('profile')
```

Comment:
[q] Any error handling here for failures - will CDK give a useful error message or just a python stack trace?

Reply:
cdk diff fails which is the first step (before cdk deploy), and notifies user of missing required properties